addons/hr_holidays/models/hr_holidays.py

```diff
@@ -504,7 +504,7 @@ def action_validate(self):
                     leaves += self.with_context(mail_notify_force_send=False).create(values)
                 # TODO is it necessary to interleave the calls?
                 leaves.action_approve()
-                if leaves[0].double_validation:
+                if leaves and leaves[0].double_validation:
```

Comment:
Is it possible to have several leaves where only some have `double_validation == True`?
If so, should probably be `leaves.filtered(lambda l: l.double_validation).action_validate()`
If not, LGTM